# External BLAS/LAPACK used, no need to build Hypre provided ones.
# Without deleting those lines, make complains about non-existing hypre provided BLAS/LAPACK object files
# B. Hajgato 04/08/2014
--- hypre-2.9.0b/src/lib/Makefile.orig	2012-05-26 01:54:08.000000000 +0200
+++ hypre-2.9.0b/src/lib/Makefile	2014-07-24 13:45:56.309096985 +0200
@@ -32,8 +32,6 @@
 STRUCTLSFILES = ${HYPRE_SRC_TOP_DIR}/struct_ls/*.o
 STRUCTMVFILES = ${HYPRE_SRC_TOP_DIR}/struct_mv/*.o
 UTILITIESFILES = ${HYPRE_SRC_TOP_DIR}/utilities/*.o
-BLASFILES = ${HYPRE_SRC_TOP_DIR}/blas/*.o
-LAPACKFILES = ${HYPRE_SRC_TOP_DIR}/lapack/*.o
 
 FILES_HYPRE = \
 $(SUPERLUFILES)\
@@ -55,9 +53,7 @@
 $(SSTRUCTMVFILES)\
 $(STRUCTLSFILES)\
 $(STRUCTMVFILES)\
-$(UTILITIESFILES)\
-$(BLASFILES)\
-$(LAPACKFILES)
+$(UTILITIESFILES)
 
 SONAME = libHYPRE-${HYPRE_RELEASE_VERSION}.so
 SOLIBS = ${MPILIBDIRS} ${MPILIBS} ${LAPACKLIBDIRS} ${LAPACKLIBS}\
@@ -103,8 +99,6 @@
 	${AR} $@ $(STRUCTLSFILES)
 	${AR} $@ $(STRUCTMVFILES)
 	${AR} $@ $(UTILITIESFILES)
-	${AR} $@ $(BLASFILES)
-	${AR} $@ $(LAPACKFILES)
 	${RANLIB} $@
 
 libHYPRE.so: ${FILES_HYPRE}
